core/Container/ContainerFactory.php

```diff
@@ -82,4 +82,16 @@ private function addEnvironmentConfig(ContainerBuilder $builder)
 
         $builder->addDefinitions($file);
     }
+
+    private function addPluginConfigs(ContainerBuilder $builder)
+    {
+        $plugins = Config::getInstance()->Plugins['Plugins'];
+
+        foreach ($plugins as $plugin) {
+            $file = Manager::getPluginsDirectory() . $plugin . '/plugin.php';
+            if (file_exists($file)) {
```

Comment:
@diosmosis haven't thought of that. Actually I'd be :+1: We always try to use same structure already (eg with other folders `Columns`, `tests/UI` and `tests/*`, `Tracker`, `Visualization`, ... ) so would make sense. I'm sure we'll have a generator for this at some point so it wouldn't matter when it is in a `config` directory. Would also allow to add maybe other config entries if needed. Maybe the `config` folder would be also useful if we allow to provide something like `config-dev.php` or `config/environment/dev.php` that will be automatically used for development etc?

`/config.php` would be otherwise ok for me as well